src/softphone.js

```diff
@@ -610,4 +610,7 @@
     };
 
     connect.SoftphoneManager = SoftphoneManager;
+
+    connect.fetchUserMedia = fetchUserMedia;
+    connect.USER_MEDIA_CONSTRAINT = USER_MEDIA_CONSTRAINT;
```

Comment:
it would be helpful to understand the exact use case here as the changes into the constraint needs changes into the function and the internal implementation!

Please go ahead make the changes you mentioned. keeping fetch user media global should be fine.